backend/webserver/util/mask_rcnn.py

```diff
@@ -18,14 +20,14 @@ class CocoConfig(Config):
     NAME = "coco"
     GPU_COUNT = 1
     IMAGES_PER_GPU = 1
-    NUM_CLASSES = 1 + 80
+    NUM_CLASSES = len(CLASS_NAMES)
```

Comment:
Shouldn't it be `1 + len(CLASS_NAMES)` to account for BG?

Reply:
I think it depends if you include the BG tag in `MASK_RCNN_CLASSES` in the compose files or not.